docs/en_us/internal/testing.rst

```diff
@@ -610,6 +616,32 @@ During acceptance test execution, Django log files are written to
 
 **Note**: The acceptance tests can *not* currently run in parallel.
 
+
+Testing internationalization with dummy translations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any text you add to the platform should be internationalized. To generate
+translations for your new strings, run the following:
+
+::
+
+    paver i18n_dummy
+
+This will generate dummy translations for the individual language files.
+
+You should then run:
+
+::
+
+    ./manage.py lms compilejsi18n
+
+To compile the translations and make them available for your local environment
+and sandbox previewing.
+
+This second command is normally run during each release. Previewing your
+translations locally won't work without first running it.
```

Comment:
@dianakhuang @nedbat Is that work something that will happen soon? If so, then I can hold off on this documentation updates until then. Otherwise, we can update this as necessary when that work is complete.